debian/landscape-client.postinst

```diff
@@ -139,6 +139,12 @@ END
             fi
             rm -f $DEFAULTS
         fi
+
+        # Since motd shows a different number of security updates,
+        # suppress the message if registered.
+        if landscape-config --is-registered >/dev/null 2>&1; then
```

Comment:
Nice try, but that won't work. The chances the client is already registered before being installed are pretty slim.

Reply:
Fair enough - I guess a second install would have to be run, which isn't going to happen any time soon.